details.py

```diff
@@ -0,0 +1,20 @@
+from django.conf import settings
+from django_mako_plus import view_function, jscontext
+from datetime import datetime, timezone
```

Comment:
I don't think you need this import statement to run your code without any errors, you could probably delete it

Reply:
True, that was just copy paste and I should get rid of it